qadence/mitigations/analog_zne.py

```diff
@@ -48,6 +64,25 @@ def analog_zne(
         KeyError(f"A noise model should be choosen from {Noise.list()}. Got {noise_model}.")
     backend = backend_factory(backend=BackendName.PULSER, diff_mode=None)
     backend = cast(Backend, backend)
+    stretches = mitigation.options.get("stretches", None)
+    # Signals to use the stretches as parameters for the ZNE data.
+    # They should be embedded before use.
+    # expression_to_uuids
+    if stretches is not None:
+        # Retrieve the parameter name and values to stretch over.
+        param_name, stretch_vals = list(stretches.items())[0]
+        # Convert to Parameter type.
+        param = Parameter(param_name)
+        # Get the map from parameters to uuids.
+        expr_to_uuids = expression_to_uuids(circuit.abstract.block)
```

Comment:
I do not really understand why you need to regenerate the `Parameter` instance here.

Reply:
Because keys to the `expr_to_uuids ` dict are `Parameter` and I only know what parameters to stretch over from their `str` name.